src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -93,7 +94,7 @@ export class QuestionsPageComponent {
   evalIfLastQuestionnaireToShowESMRatingQuestion(currentQuestionId) {
     const time = new Date()
     if (this.questionTitle === 'ESM' && currentQuestionId === 'esm_beep') {
-      if (time.getHours() >= 19) {
+      if (time.getHours() >= 21) {
```

Comment:
It may not be exactly 21 (could be a bit less too). In statging project its 21 but other studies have different values. Just stick to 19 for now.